cloud/src/meta-service/meta_service.cpp

```diff
@@ -2447,6 +2447,12 @@ void MetaServiceImpl::get_delete_bitmap_update_lock(google::protobuf::RpcControl
                     return;
                 }
             } else {
+                // initiator may remain in initiators when upgrade ms, need to clear it
+                if (lock_info.initiators_size() > 0) {
+                    lock_info.clear_initiators();
```

Comment:
for request->lock_id() == COMPACTION_DELETE_BITMAP_LOCK_ID, the lock key is not put into fdb